docs/contracts.rst

```diff
@@ -862,14 +876,17 @@ Methods
         nodes would result in an error like:  ``ValueError: {'code': -32602, 'message': 'too many arguments, want at most 1'}``
 
 .. py:method:: ContractFunction.buildTransaction(transaction)
+    .. warning:: Deprecated: This method is deprecated in favor of :class:`~build_transaction`
```

Comment:
I think there needs another line between 878 and 879 as well.

Reply:
fixed